UnitsNet.Tests/CustomCode/RotationalSpeedTests.cs

```diff
@@ -42,6 +42,15 @@ public class RotationalSpeedTests : RotationalSpeedTestsBase
 
         protected override double RevolutionsPerSecondInOneRadianPerSecond => 1/(2*Math.PI);
 
+        protected override double DegreesPerSecondInOneRadianPerSecond => (180 / Math.PI) * 1;
+
+        protected override double MillidegreesPerSecondInOneRadianPerSecond => ((180 / Math.PI) * 1)* 1E3;
+
```

Comment:
Should be fixed in commit  e6ba20a.
You are right, I found myself wondering why I was repeating the formula...
Thanks for the tip.